lib/readwise/client.rb

```diff
@@ -8,6 +8,8 @@ module Readwise
   class Client
     class Error < StandardError; end
 
+    BASE_URL = "https://readwise.io/api/v2/"
```

Comment:
Should this be shared any way other than a class constant?

Reply:
I think for this stage, this is a fine place.